packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -1094,6 +1093,21 @@ describe('function', () => {
       expect(() => { fn.grantInvoke(new iam.ServicePrincipal('elasticloadbalancing.amazonaws.com')); })
         .toThrow(/Cannot modify permission to lambda function/);
     });
+
+    test('on an imported function (different account & w/ skipPermissions', () => {
+      // GIVEN
+      const stack = new cdk.Stack(undefined, undefined, {
+        env: { account: '111111111111' }, // Different account
+      });
+      const fn = lambda.Function.fromFunctionAttributes(stack, 'Function', {
+        functionArn: 'arn:aws:lambda:us-east-1:123456789012:function:MyFn',
+        skipPermissions: true,
+      });
+
+      // THEN
+      expect(() => { fn.grantInvoke(new iam.ServicePrincipal('elasticloadbalancing.amazonaws.com')); })
+        .not.toThrow();
```

Comment:
What's with the weird formatting here...?

```suggestion
      expect(() => {
        fn.grantInvoke(new iam.ServicePrincipal('elasticloadbalancing.amazonaws.com'));
      }).not.toThrow();
```